src/super_gradients/training/metrics/user_metrics/README.md

```diff
@@ -0,0 +1,64 @@
+<div "center">
+  <img src="docs/assets/SG_img/SG - Horizontal.png" width="600"/>
+ <br/><br/>
+
+## Introduction
+This page demonstrates how you can register your own metrics, so that SuperGradients can access it with a name `str`, for
+example, when training from a recipe config.
+
+## Usage
+**In your python script**
+1. Define your PyTorch metric (`torchmetrics.Metric`) in the new module.
+2. Import the `@register` decorator 
```

Comment:
@register_metric

Reply:
Updated, there are also other changes in the ReadMe